README.rst

```diff
@@ -76,6 +76,9 @@ Setup redis connection and enable caching for desired models:
     # should be compatible or subclass cacheops.redis.CacheopsRedis
     CACHEOPS_CLIENT_CLASS = 'your.redis.ClientClass'
 
+    # You can use two pickling libs: pickle or dill
+    CACHEOPS_PICKLE_LIB = 'pickle'
```

Comment:
I would call it `CACHEOPS_SERIALIZER` and make it support both string and object:

```python
CACHEOPS_SERIALIZER = "pickle"       # default
CACHEOPS_SERIALIZER = "dill"         # use dill
CACHEOPS_SERIALIZER = "your.module"  # use any module with .loads() and .dumps()


# Custom implementation, small and promitive enough to inline in settings
import pickle

class Serializer:
    dumps = lambda obj: pickle.dumps(obj, 3)  # Use non-default protocol
    loads = pickle.loads

CACHEOPS_SERIALIZER = Serializer  
```

Reply:
Also, give it a separate section in README.